_includes/v22.2/ui/sessions.md

```diff
@@ -6,24 +6,27 @@ To filter the sessions, click the **Filters** field.
 
 <img src="{{ 'images/v22.2/ui-session-filter.png' | relative_url }}" alt="Session filter" />
 
-To filter by [application]({{ link_prefix }}connection-parameters.html#additional-connection-parameters), select **App** and choose one or more applications.
+- To filter by [application]({{ link_prefix }}connection-parameters.html#additional-connection-parameters), select **App** and choose one or more applications.
 
-- Queries from the SQL shell are displayed under the `$ cockroach` app.
-- If you haven't set `application_name` in a client connection string, it appears as `unset`.
+    - Queries from the SQL shell are displayed under the `$ cockroach` app.
+    - If you haven't set `application_name` in a client connection string, it appears as `unset`.
 
-To filter by session duration, specify the session time and unit.
+- To filter by username or session status, select **Username** or **Session Status** and choose one or more checkboxes.
```

Comment:
Nit: User name 

<img width="146" alt="image" src="https://user-images.githubusercontent.com/81193618/189795310-94cf0d0b-eef9-4645-8be4-f6690ad6ff4e.png">

Reply:
@kevin-v-ngo The checkbox in the filter is labeled **Username**. See the screenshot ui-session-filter.png in this PR.

Reply:
@kevin-v-ngo  I filed this: https://github.com/cockroachdb/cockroach/issues/87960